test/lit/passes/asyncify_pass-arg=asyncify-asserts_pass-arg=asyncify-onlylist@waka.wast

```diff
@@ -39,128 +39,34 @@
   ;; CHECK:      (export "asyncify_get_state" (func $asyncify_get_state))
 
   ;; CHECK:      (func $calls-import
-  ;; CHECK-NEXT:  (local $0 i32)
-  ;; CHECK-NEXT:  (local.set $0
-  ;; CHECK-NEXT:   (global.get $__asyncify_state)
-  ;; CHECK-NEXT:  )
-  ;; CHECK-NEXT:  (block
-  ;; CHECK-NEXT:   (call $import)
-  ;; CHECK-NEXT:   (if
-  ;; CHECK-NEXT:    (i32.ne
-  ;; CHECK-NEXT:     (global.get $__asyncify_state)
-  ;; CHECK-NEXT:     (local.get $0)
-  ;; CHECK-NEXT:    )
-  ;; CHECK-NEXT:    (unreachable)
-  ;; CHECK-NEXT:   )
-  ;; CHECK-NEXT:  )
+  ;; CHECK-NEXT:  (call $import)
```

Comment:
This seems wrong - this calls an import, so asyncify should instrument it, as it did before.

Reply:
@kripken As far I undestand in ModuleAnalyzer.map this function stored as **'import'**, but when InstumentedProxy.runOnFunction is called the function name is **'calls-import'**, this is the reason why it is not instrumented.

Reply:
If you run it in verbose mode you will see following output:
```
[asyncify] import is an import that can change the state
[asyncify] import2 is an import that can change the state
[asyncify] import3 is an import that can change the state
[asyncify] calls-indirect can change the state due to initial scan
[asyncify] calls-import can change the state due to import
[asyncify] calls-import2-drop can change the state due to import2
[asyncify] returns can change the state due to import2
[asyncify] calls-import's state is set based on the only-list to 0
[asyncify] calls-import2-drop's state is set based on the only-list to 0
[asyncify] returns's state is set based on the only-list to 0
[asyncify] calls-indirect's state is set based on the only-list to 0
```

Initially it marked as instrumented, but becaouse of asyncify-only-list it's marked back to non instrumented.

```
[asyncify] calls-import can change the state due to import
[asyncify] calls-import's state is set based on the only-list to 0
```

Because of that InstrumentedProxy didn't run passes on it.

Reply:
I see, thanks - I missed that there is an only-list in this test.